### PR TITLE
SPCTR-2472 - New: Implement SureCart Licensing Admin Interface for Spectra Pro.

### PR DESCRIPTION
**Pull request by RajkiranBagal**
### Description
https://brainstormforce.atlassian.net/browse/SPCTR-2472?atlOrigin=eyJpIjoiZGZkZjkwMzY3NmUzNDAzYmEwOWMwNjQ3NTk5MGMxMzUiLCJwIjoiaiJ9

### Types of changes
New feature (non-breaking change which adds functionality) 

### How has this been tested?
- In the card.

### Checklist:
- [x] My code is tested
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: admin-core/assets/src/dashboard-app/pages/settings/MyAccount.js [hunk 1]
```diff
@@ -1,103 +1,155 @@
 import React, {useState} from 'react';
```
Comment (adi3890) [inline admin-core/assets/src/dashboard-app/pages/settings/MyAccount.js:1]: can we use wordpress/element

File: admin-core/assets/src/dashboard-app/pages/settings/MyAccount.js [hunk 2]
```diff
@@ -1,103 +1,155 @@
...
+					<div className="w-9/12">
+						{ ! licenseStatus && (
 							<p
 								style={ {
 									margin: '0.5rem 0 0', /* Replicating Tailwind: mt-2 */
 									color: '#64748b', /* Replicating Tailwind: text-slate-500 */
 								} }
 							>
-								{ licenseMessage }
+								{ __( 'Activate ', 'ultimate-addons-for-gutenberg' ) }
```
Comment (adi3890) [inline admin-core/assets/src/dashboard-app/pages/settings/MyAccount.js:75]: can we use sprint here to provide better translatable sentence

Comment (RajkiranBagal) [reply admin-core/assets/src/dashboard-app/pages/settings/MyAccount.js:75]: It is a mix of markup & strings so did not use sprintf, Translated the missing string to translate.

File: admin-core/assets/src/dashboard-app/pages/settings/MyAccount.js [hunk 3]
```diff
@@ -1,105 +1,171 @@
-import React, {useState} from 'react';
+import { useState, } from '@wordpress/element';
```
Comment (adi3890) [inline admin-core/assets/src/dashboard-app/pages/settings/MyAccount.js:1]: seems like we missed to remove comma

File: admin-core/assets/src/dashboard-app/pages/settings/MyAccount.js (hunk 2 above)
Comment (adi3890) [reply admin-core/assets/src/dashboard-app/pages/settings/MyAccount.js:89]: @zane has implemented similar case in one of his PRs. Can you please try that

Comment (adi3890) [reply admin-core/assets/src/dashboard-app/pages/settings/MyAccount.js:89]: Oh Sorry @zane 
I wanted to tag @bsf-zanev

**Review by adi3890 — APPROVED** [review]

**Pull request merged**